src/patternfly/components/Touchspin/examples/Touchspin.md

```diff
@@ -0,0 +1,140 @@
+---
+id: Touch spin
+section: components
+cssPrefix: pf-c-touchspin
```

Comment:
Is this called "Touchspin" or "Touch spin"? If it's two words, I think there should be a hyphen in the component name.

Reply:
Confirmed with @mcarrano it's "Touchspin"

Reply:
yep thanks for clarifying that was a mistake